packages/node-addon-examples/tests/buffers/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "buffers-test",
+  "version": "0.0.0",
+  "description": "Tests of runtime buffer functions",
+  "main": "addon.js",
+  "private": true,
+  "dependencies": {
+    "bindings": "~1.5.0"
+  },
+  "scripts": {
+    "test": "node --expose-gc addon.js"
+  },
+  "gypfile": true,
+  "readme": "ERROR: No README data found!",
+  "_id": "function_arguments@0.0.0"
```

Comment:
What's this? If this was copied from somewhere, we should make sure to attribute accordingly 👍